### ISSUE TITLE
Question: When you click "Submit Results", where do the results go?

### ISSUE THREAD

**Issue by carmacleod** [w3c/aria-at#390]
I'm having trouble finding where the test results are stored.
I'd like to look at the results for the Editor Menubar on VoiceOver/macOS, because I think there are some serious VoiceOver issues that need reporting, and I was hoping to point to this project's results for menubar testing. :)
Specifically,
- menuitemradio role is not conveyed (so user has no idea unchecked radio is checkable, or that they're in a radio group)
- menuitemcheckbox role is not conveyed (so user has no idea unchecked checkbox is checkable)
- label of first menuitem with focus is "sometimes" not conveyed when submenu is opened (so user doesn't know that focus has moved to the first menuitem, and if they type down arrow again, they will move to the second menuitem)
- weird rendering of the number "1" in label of checked menuitemradio and menuitemcheckbox

In other words, I would expect to see test failures for:
- https://w3c.github.io/aria-at/tests/menubar-editor/test-12-open-submenu-of-menubar-interaction.html?at=voiceover_macos
- https://w3c.github.io/aria-at/tests/menubar-editor/test-28-read-unchecked-menuitemradio-in-a-group-in-a-submenu-interaction.html?at=voiceover_macos
- and several other voiceover/macos tests from https://w3c.github.io/aria-at/tests/menubar-editor/

But I don't know where to look. :)

**Comment by jesdaigle** [comment]
@carmacleod there is a web app for test results: https://aria-at.w3.org/reports/test-plans/6 

It is still in it's younger stages and being built, but perhaps you will find this helpful. :)
